core/src/main/java/io/grpc/internal/DelayedClientTransport.java

```diff
@@ -183,6 +184,7 @@ private PendingStream createPendingStream(
     if (getPendingStreamsCount() == 1) {
       syncContext.executeLater(reportTransportInUse);
     }
+    Arrays.stream(tracers).forEach(ClientStreamTracer::createPendingStream);
```

Comment:
This shouldn't be allowed on Android, as it was added in API level 24. I seems animalsniffer is broken.